### PR TITLE
Added link whitelist functionality for redbrick use case

### PR DESCRIPTION
**Pull request by lukebarton**

### CODE REVIEW

File: tasks/writeScenarioData.js [hunk 1]
```diff
@@ -105,6 +105,9 @@ module.exports = function (grunt) {
     return _.map(scenario, function (resource) {
       var resourceClone = _.cloneDeep(resource);
       if (resourceClone.response) {
+        if (resourceClone.linksWhitelist) {
```
Comment (afternoon) [inline tasks/writeScenarioData.js:108]: `linksWhitelist` is a bit wordy. How about `rels`?

File: tasks/writeScenarioData.js [hunk 2]
```diff
@@ -105,6 +105,9 @@ module.exports = function (grunt) {
...
+          links = _.pick(links, resourceClone.linksWhitelist);
```
Comment (afternoon) [inline tasks/writeScenarioData.js:109]: Does this overwrite the value of links for later iterations? How about:

```
if (resourceClone.linksWhitelist) {
  resourceClone.response._links = _.pick(links, resourceClone.linksWhitelist);
}
else {
  resourceClone.response._links = links;
}
```

Comment (lukebarton) [inline tasks/writeScenarioData.js:109]: Nope:
http://jsfiddle.net/wYPat/

```
var foo = 'bar';
var fn = function(foo) {
  foo = 'baz';
  return foo;
}

console.log(foo === 'bar');
console.log(fn(foo) === 'baz');
console.log(foo === 'bar'); // foo !== 'baz'
```

Output:

```
true
true
true
```

**Comment by afternoon** [comment]
Suggested changes made by @mike-tempest in PR #2.
